tests/champion.test.ts

```diff
@@ -54,4 +54,9 @@ describe('Test champion fetching.', () => {
       'https://raw.communitydragon.org/pbe/plugins/rcp-be-lol-game-data/global/default/assets/characters/kayn/skins/base/kaynloadscreen.jpg'
     );
   });
+
+  test('Check champion caching', async () => {
+    expect(client.champions.cache.firstKey()).toBe(kayn.name);
```

Comment:
instead of using `firstKey()` and `first()`, you should be using `has(key)` and `get(key)` to check.

Even though this works too, lets use the intended methods for these things.